setup.py

```diff
@@ -13,9 +13,9 @@ def read_file(name):
 
 
 install_requires = [
-    "requests==2.24.0",
-    "ujson==1.35",
-    "google-cloud-bigquery==1.27.2",
+    "requests==2.25.0",
+    "ujson==4.0.1",
+    "google-cloud-bigquery==2.5.0",
```

Comment:
We shouldn't be pinning these to specific versions but should instead do ranges or minimum support or something like that.

This might be better?:

```
requests>=2.24.0<3.0.0
ujson>=4.0.0<5.0.0
google-cloud-bigquery>=2.5.0<3.0.0
```

I'm guessing wildly on the version promises for those projects and assuming that they're semver-ish.

Reply:
Yes, definitely. Feel free to change that as part of this PR, if you like.

Reply:
Sounds good to me